### DISCUSSION TITLE
how to create a new conversation

### DISCUSSION THREAD

**Discussion by clayouuz** [community/community#56592] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

<img width="534" alt="image" src="https://github.com/community/community/assets/115384870/e3d4a528-fb5b-4b46-ba0c-51fd1056091c">  

i have some bad prompts which lead to bad requests. i want to clear them but failed.  

what can i do to create a new conversation?

**Comment by davecheney** [comment, marked as answer]
![Screenshot 2023-05-29 at 11 37 15](https://github.com/community/community/assets/7171/e8d5acf4-055e-4641-bfda-f0da3049a64f)
Push this button.
